draft-packages/button/docs/IconButton.stories.tsx

```diff
@@ -1,179 +0,0 @@
-import configureIcon from "@kaizen/component-library/icons/configure.icon.svg"
-import * as React from "react"
-import { IconButton } from ".."
-import { CATEGORIES, SUB_CATEGORIES } from "../../../storybook/constants"
-
-export default {
-  title: `${CATEGORIES.components}/${SUB_CATEGORIES.button}/Icon Button`,
-  component: IconButton,
-  parameters: {
-    docs: {
-      description: {
-        component: 'import { IconButton } from "@kaizen/draft-button";',
```

Comment:
This bit of documentation tells users how to import the component.

It would be nice to copy paste this line so that the new combined story documents how to import both `Button` and `IconButton`.

Or show a combined import:

```
'import { Button, IconButton } from "@kaizen/draft-button";',
```

Reply:
Updated to have combined import description. Thanks!